modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -1,114 +1,83 @@
-+
-.Prometheus configuration
-====
-[source,yaml,subs="+quotes,+attributes,+macros"]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: prometheus-config
-  namespace: monitoring
-data:
-  prometheus.yml: |-
-      global:
-        scrape_interval: 5s <1>
-        evaluation_interval: 5s <2>
-      scrape_configs: <3>
-        - job_name: 'DevWorkspace'
-          scheme: https
-          authorization:
-            type: Bearer
-            credentials_file: '/var/run/secrets/kubernetes.io/serviceaccount/token'
-          tls_config:
-            insecure_skip_verify: true
-          static_configs:
-            - targets: ['devworkspace-controller-metrics.__<DWO_{orch-namespace}>__:8443'] <4>
-        - job_name: 'DevWorkspace webhooks'
-          scheme: https
-          authorization:
-            type: Bearer
-            credentials_file: '/var/run/secrets/kubernetes.io/serviceaccount/token'
-          tls_config:
-            insecure_skip_verify: true
-          static_configs:
-            - targets: ['devworkspace-webhookserver.__<DWO_{orch-namespace}>__:9443'] <5>
-----
-<1> The rate at which a target is scraped.
-<2> The rate at which the recording and alerting rules are re-checked.
-<3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.
-<4> The scrape target for the metrics from port `8443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
-<5> The scrape target for the metrics from port `9443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
-====
-
-. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
+. Allow the in-cluster Prometheus instance to detect the ServiceMonitor in the {prod-short} namespace. The default {prod-short} namespace is `{prod-namespace}`.
 +
 [source,terminal,subs="+attributes,quotes"]
 ----
-$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+oc label namespace {prod-namespace} openshift.io/cluster-monitoring=true
 ----
 
 .Verification
 
-. Use port forwarding to access the `Prometheus` Service locally:
+. For a fresh installation of {prod-short}, generate metrics by creating a {prod-short} workspace from the Dashboard.
+
+. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Metrics*.
+
+. Run a PromQL query to confirm that the metrics are available. For example, enter `devworkspace_started_total` and click *Run queries*.
 +
-[source,terminal,subs="+attributes,quotes"]
+For more metrics, see xref:ref_devworkspace-specific-metrics[].
+
+[TIP]
```

Comment:
The preview is showing some AsciiDoc glitch, so I recommend adding empty lines within the TIP text or just copy-paste what you have in `proc_collecting-che-metrics-with-prometheus.adoc` on lines `110-128`.

Reply:
I've added a new commit 👍